translations/extract.py

```diff
@@ -18,7 +18,7 @@
 Expects to be run from the root directory of the repository.
 
 Usage:
-   extract.py pkg/kubectl/cmd/apply.go
+   ./extract.py ../pkg/kubectl/cmd/apply/apply.go
```

Comment:
can't this just be called from the parent directory like so?
`./translations/extract.py ./pkg/kubectl....`

Reply:
this change fails CI:
https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/74402/pull-kubernetes-verify/123392

i suggest we lave it the way it is.
thanks

/close